Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,69 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * copyAction event handler, launch the universal discovery widget
+         * to choose a location to copy the content
+         *
+         * @method _copySelectLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _copySelectLocation: function (e) {
+            this.fire('contentDiscover', {
+                config: {
+                    title: "Select the location you want to copy your content into",
+                    contentDiscoveredHandler: Y.bind(this._copyContent, this),
+                },
+            });
+        },
+
+        /**
+         * Copy the content to the selected location
+         *
+         * @method _copyContent
+         * @protected
+         * @param {EventFacade} e
+         */
+        _copyContent: function (e) {
+            var app = this.get('app'),
+                parentLocationId = e.selection.location.get('id'),
+                locationId = this.get('location').get('id'),
+                that = this,
+                notificationIdentifier =  'copy-notification-' + parentLocationId + '-' + locationId;
+
+            this._fireNotification('The copy operation of your content has begun', notificationIdentifier, 'started', 5);
```

Comment:
same remark as in https://github.com/ezsystems/PlatformUIBundle/pull/229#discussion_r31312761, the message can be improved to contain the name of the content being copied and the parent content name as well.